projenrc/issue-labeler.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, github } from 'projen';
+import { JobPermission } from 'projen/lib/github/workflows-model';
+import { TypeScriptProject } from 'projen/lib/typescript';
+
+const OSDS_DEVS = ['ashishdhingra', 'khushail', 'hunhsieh'];
+const AREA_AFFIXES = ['@aws-cdk/'];
+const AREA_PARAMS = [
+  { area: '@aws-cdk/cli-lib-alpha', keywords: ['cli', 'cli-lib', 'cli-lib-alpha'], labels: ['@aws-cdk/cli-lib-alpha'] },
+  { area: '@aws-cdk/cloud-assembly-schema', keywords: ['cloud-assembly', 'schema'], labels: ['@aws-cdk/cloud-assembly-schema'] },
+  { area: '@aws-cdk/cloudformation-diff', keywords: ['diff', 'cloudformation'], labels: ['@aws-cdk/cloudformation-diff'] },
+  { area: '@aws-cdk/toolkit-lib', keywords: ['toolkit', 'programmtic toolkit', 'toolkit-lib'], labels: ['@aws-cdk/toolkit-lib'] },
+  { area: 'aws-cdk', keywords: ['aws-cdk', 'cli', 'cdk cli'], labels: ['aws-cdk'] },
+  { area: 'cdk-assets', keywords: ['assets', 'cdk-assets'], labels: ['cdk-assets'] },
+];
+
+/**
+ * See https://github.com/aws-github-ops/aws-issue-triage-manager
+ */
+interface TriageManagerOptions {
+  target: 'pull-requests' | 'issues' | 'both';
+  excludedExpressions?: string[];
+  includedExpressions?: string[];
+  includedLabels?: string[];
+  excludedLabels?: string[];
+  defaultArea?: string;
+  parameters?: string;
+  affixes?: string;
+  areaIsKeyword?: boolean;
+  needEnvs?: boolean;
```

Comment:
How come one jobs needs it and one doesn't? I see its [configured on the workflow level](https://github.com/aws/aws-cdk/blob/main/.github/workflows/issue-label-assign.yml#L58) in the aws-cdk repo.

Reply:
its configured on the workflow level, yes. but it doesn't look like the other job uses those env variables, even if it might have access to it.

Reply:
i didn't add the guidance step, which would utilize the workflow level env vars (probs why they put it on the workflow level).

however, projen doesn't currently support workflow-level env variables, so if we were to include the guidance job i'd just add `needEnvs: true` to that step for now (until we have projen support).

Reply:
Followup then - do we not need the guidance step?